htdocs/core/modules/modSociete.class.php

```diff
@@ -281,7 +281,7 @@ function __construct($db)
 		$this->export_sql_end[$r] .=' LEFT JOIN '.MAIN_DB_PREFIX.'c_payment_term as payterm ON s.cond_reglement = payterm.rowid';
 		$this->export_sql_end[$r] .=' LEFT JOIN '.MAIN_DB_PREFIX.'c_paiement as paymode ON s.mode_reglement = paymode.id';
 		$this->export_sql_end[$r] .=' WHERE s.entity IN ('.getEntity('societe', 1).')';
-		if(!$user->rights->societe->client->voir) $this->export_sql_end[$r] .=' AND sc.fk_user = '.$user->id;
+		if(!$user->rights->societe->client->voir) $this->export_sql_end[$r] .=' AND (sc.fk_user = '.$user->id.' OR sc.fk_user IN (SELECT rowid FROM '.MAIN_DB_PREFIX.'user where fk_user= '.$user->id.'))';
```

Comment:
Why limiting to firt level of subordinates.
You can make 
$subordinatesids = $user->getAllChildIds()
and make into select
count(subronidatesids) ? ' OR sc.fk_user IN ('.$subronidatesids.')' : '';

Also, the result return something not consistent with other screens. The export return also third of subordinates but not all other pages. So we must introduce a hidden constant "THIRDPARTY_RECURSIVE_SALE_REPRESENTATIVES" option to start to introduce this test in code.